### PR TITLE
docs: Rudder/pedal instructions

### PR DESCRIPTION
**Pull request by tohoku**
## Summary
Having spent a lot of time trying to get to the bottom of my rudder pedal settings, I suggest adding a page to the FBW documentation page to save other people the headache. Initially I thought this was a problem with the FBW A320 flight model which stopped me from taxiing properly.

The FS2020 forums contain a lot of contradicting information, and I have yet to find a post which describes the correct settings.

I am hoping others can chime in with settings for other pedals than the Thrustmaster ones.

### Location

docs/fbw-a32nx/rudder.md
docs/fbw-a32nx/assets/rudder/*.jpg

Discord username (if different from GitHub):

### CODE REVIEW

File: docs/fbw-a32nx/rudder.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+![Rudder Control Settings](assets/rudder/ThrustmasterRudderSettings.jpg "Rudder Control Settings"){loading=lazy}
+
+### 2. Confirm Calibration
+
+Check that the mappings are correct. If you push the pedals forwards ("toe brakes"), the Joystick L-Axis X and Joystick L-Axis Y indicators should move from top right (0% brakes applied) to bottom left (100% brakes applied).
+
+![Rudder Sensitivity Settings](assets/rudder/ThrustmasterRudderSensitiviy.jpg "Rudder Sensitivity Settings"){loading=lazy}
+
+If the Joystick L-Axis X/Y indicators only move from the center of the graphs to the left bottom, you will need to calibrate your pedals.
```
Comment (Sleinmaster) [inline docs/fbw-a32nx/rudder.md]: Should probably apply for any state where it doesn't go from 0 to 100, no matter the direction.

**Review by Sleinmaster — CHANGES_REQUESTED** [review]

**Comment by frankkopp** [comment]
I'm not really sure we need this. The A32NX does NOT need any special rudder configuration.
It works as any other aircraft in the sim in regards to rudder. 
Only difference is to disable Assistance Functions (Auto Rudder) which should be off anyway as per our recommendations.
Even when enabling the Tiller the configuration of the rudder still works but will be limited to 6° of turn (see Tiller docs). 

I would rather not start documenting standard MSFS stuff. 
If at all this could be under Support Reported Issues (https://docs-git-fork-tohoku-ruddersettings-flybywire.vercel.app/fbw-a32nx/support/reported-issues/) as a solution to rudder not working properly even if Assistance is turned off.

**Comment by Valastiri** [comment]
I think there is a use case for this if it lives under recommended settings. If we can agree on that I'll provide further review as it's not setup appropriately currently anyways.

We do recommend some basic settings be enabled for rudder pedal compatibility but that's for racing pedals. I don't mind further documentation as just makes us more complete in a sense.

> Think along the lines of "additional considerations"

**Comment by frankkopp** [comment]
> I think there is a use case for this if it lives under recommended settings. If we can agree on that I'll provide further review as it's not setup appropriately currently anyways.
> 
> 
> 
> We do recommend some basic settings be enabled for rudder pedal compatibility but that's for racing pedals. I don't mind further documentation as just makes us more complete in a sense.
> 
> 
> 
> > Think along the lines of "additional considerations" 

I see it more under the issue section as normally this the default setting anyway. And recommended settings document thing which are not default. Might even confuse people there.

**Comment by Valastiri** [comment]
> > I think there is a use case for this if it lives under recommended settings. If we can agree on that I'll provide further review as it's not setup appropriately currently anyways.
> > We do recommend some basic settings be enabled for rudder pedal compatibility but that's for racing pedals. I don't mind further documentation as just makes us more complete in a sense.
> > > Think along the lines of "additional considerations"
> 
> I see it more under the issue section as normally this the default setting anyway. And recommended settings document thing which are not default. Might even confuse people there.

Oh if it's more of a default setting then yea issues would be more appropriate. However, would be pretty bulky considering how much text is in this PR in general.

@tohoku Give us a bit to consider where is the best place to live but thanks for the PR nonetheless.

**Comment by tohoku** [comment]
Everyone, thank you for your kind feedback,

I should probably tell you the reason why I went down this rabbit hole in the first place: I have been an enthusiast since MSFS 5.0, but I only installed the FBW A32NX for the first time last week.

I immediately ran into problems with taxiing. I could push back and start taxiing, but as soon as I hit the brakes I couldn't get the plane to move again. Initially I thought there was a problem with the FBW flight model and uninstalled all add-ons, but the problem consisted. (I did check that I had all the Assistance Functions set to off.)

Then I switched back to the default Asobo A320 but did not notice any problems with taxiing. (It seems like the Asobo A320 is more forgiving with the toe brake settings when taxiing?)

I can put the info in this PR up on a forum, but my impression from reading all the MSFS forums is that this (taxiing and toe brakes) is a challenge people mostly run into after moving from the Asobo A320 to a more advanced plane.

Based on the above, I would argue that the lack of concise info out there may be stopping new users from enjoying the FBW A32NX, leading them to give up.

**Comment by Valastiri** [comment]
@tohoku We ran this buy our support team for some additional input. I believe there's a general consensus to add this into the docs but not where it is currently. 

There were two choices:

1. As an issue in the reported issues with a link to the pages you made.
2. As a subpage or included in the `Common Hardware` section in the API and Hardware as an additional reference. 

We are leaning towards the second choice. If you need help setting this up please let us know or we can access the PR and move the files around for you.

File: docs/fbw-a32nx/rudder.md (hunk 1 above)
Comment (tohoku) [reply docs/fbw-a32nx/rudder.md]: @Sleinmaster The instructions have been updated to be more generic.

**Comment by tohoku** [comment]
@Valastiri - thanks for the suggestions. I have updated the page as suggested in alternative 2 with a subpage for detailed settings.

rudder-update

**Comment by Valastiri** [comment]
> @Valastiri - thanks for the suggestions. I have updated the page as suggested in alternative 2 with a subpage for detailed settings.
> 
> rudder-update

That's great -- we're still going through some discussion on where exactly this should live and if it's alright with you a maintainer might move things around for ya. Just so we logically have this in the right place.

**Comment by tohoku** [comment]
@Valastiri Sure, no worries at all.

**Comment by frankkopp** [comment]
I moved the page to the support section and added a "Rudder Operation Issue".

image

image

@tohoku could you please double check all your images - her are some links to discord for how to do this:
https://discord.com/channels/738864299392630914/791494566058000434/907228632206962738
https://discord.com/channels/738864299392630914/791494566058000434/1121099138369605723

**Comment by tohoku** [comment]
@frankkopp Thanks for the directions. I have updated the images to be max 826px wide and < 100 KB.

**Comment by tohoku** [comment]
One more thing, maybe adding a link to the "Rudder Operation Issues" section on the support page from the "Unable to Move and Taxi" section is appropriate?

https://docs-pivophukd-flybywire.vercel.app/fbw-a32nx/support/reported-issues/#unable-to-move-or-taxi

**Comment by frankkopp** [comment]
> One more thing, maybe adding a link to the "Rudder Operation Issues" section on the support page from the "Unable to Move and Taxi" section is appropriate?
> 
> [docs-pivophukd-flybywire.vercel.app/fbw-a32nx/support/reported-issues/#unable-to-move-or-taxi](https://docs-pivophukd-flybywire.vercel.app/fbw-a32nx/support/reported-issues/#unable-to-move-or-taxi)

Good idea - was thinking about this as well. Go ahead!

Thanks for your contribution - really well done.

File: docs/fbw-a32nx/support/detail-pages/rudder.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+### 2. Confirm Calibration
+
+Check that the mappings are correct. If you push the pedals forwards ("toe brakes"), the Joystick L-Axis X and Joystick L-Axis Y indicators should move from top right (0% brakes applied) to bottom left (100% brakes applied). If the Joystick L-Axis X/Y indicators behave differently, you will need to calibrate your pedals.
+
+![Rudder Sensitivity Settings](../../assets/rudder/ThrustmasterRudderSensitiviy.jpg "Rudder Sensitivity Settings"){loading=lazy}
+
+### 3. Reset and Calibrate (Optional)
+
+1. From the Windows Start menu or the Control Panel, search for **`Set up USB game controllers`**.
+2. Your game controllers should be displayed. Select your pedals (i.e. **`T-Rudder`**) from the list. Click the **`Properties`** button. ![USB Controller Settings](../../assets/rudder/GameControllersThrustmaster.jpg "USB Controller Settings"){loading=lazy}
```
Comment (Sleinmaster) [inline docs/fbw-a32nx/support/detail-pages/rudder.md]:
```suggestion
2. Your game controllers should be displayed. Select your pedals (i.e., **`T-Rudder`**) from the list. Click the **`Properties`** button. ![USB Controller Settings](../../assets/rudder/GameControllersThrustmaster.jpg "USB Controller Settings"){loading=lazy}
```

File: docs/fbw-a32nx/support/detail-pages/rudder.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+3. From the **`Settings`** tab, click on **`Reset to default`**.
+4. Click on the **`Test`** tab to confirm the settings. The cursor should be displayed in the bottom right corner (0% brakes applied) or top left corner (100% brakes applied), while the Z-Axis indicator should be at 50% if the rudder is centered. ![Calibrated Settings](../../assets/rudder/GameControllersRudderSettings.jpg "Calibrated Settings"){loading=lazy}
```
Comment (Sleinmaster) [inline docs/fbw-a32nx/support/detail-pages/rudder.md]:
```suggestion
4. Click on the **`Test`** tab to confirm the settings. The cursor should be displayed in the bottom-right corner (0% brakes applied) or top-left corner (100% brakes applied), while the Z-Axis indicator should be at 50% if the rudder is centered. ![Calibrated Settings](../../assets/rudder/GameControllersRudderSettings.jpg "Calibrated Settings"){loading=lazy}
```

**Review by Sleinmaster — COMMENTED** [review]
Some small corrections and then it's good to go from my side.

**Comment by tohoku** [comment]
> > One more thing, maybe adding a link to the "Rudder Operation Issues" section on the support page from the "Unable to Move and Taxi" section is appropriate?
> > [docs-pivophukd-flybywire.vercel.app/fbw-a32nx/support/reported-issues/#unable-to-move-or-taxi](https://docs-pivophukd-flybywire.vercel.app/fbw-a32nx/support/reported-issues/#unable-to-move-or-taxi)
> 
> Good idea - was thinking about this as well. Go ahead!
> 
> Thanks for your contribution - really well done.

@frankkopp Thank you. I am just happy to contribute to the FS2020 and FBW community!

**Comment by Valastiri** [comment]
check build

**Comment by Valastiri** [comment]
lgtm - was just checking due to the amount of moving files around in case anything was broken. 

If there are no other issues will merge today and thanks for the PR!

**Pull request merged**